components/net/http_loader.rs

```diff
@@ -675,6 +675,12 @@ pub fn http_redirect_fetch(
         .unwrap()
         .set_attribute(ResourceAttribute::FetchStart);
 
+    context
+        .timing
+        .lock()
+        .unwrap()
+        .set_attribute(ResourceAttribute::DomainLookupStart);
```

Comment:
We should move this to [right before](https://github.com/servo/servo/blob/834d3193a2ed016093b6b997d29df5a607715956/components/net/http_loader.rs#L383) the HTTP request is initiated instead.

Reply:
Thanks for quick feedback!
I've moved the code as requested.